Emby.Server.Implementations/LiveTv/EmbyTV/EmbyTV.cs

```diff
@@ -2192,7 +2192,8 @@ private bool ShouldCancelTimerForSeriesTimer(SeriesTimerInfo seriesTimer, TimerI
 
         private void HandleDuplicateShowIds(List<TimerInfo> timers)
         {
-            foreach (var timer in timers.Skip(1))
+            // sort showings by HD channels first, then by startDate, record earliest showing possible
+            foreach (var timer in timers.OrderByDescending(t => Convert.ToInt32(this._liveTvManager.GetLiveTvChannel(t, this).IsHD)).ThenBy(t => t.StartDate).Skip(1))
```

Comment:
Why convert it to int?

Reply:
I wasn't sure how booleans would be sorted, but the convert had a defined output.

Reply:
False is equivalent to 0 and true to 1. No need to do the conversion

Reply:
Updated and squashed.